blog: upload-chaos-mesh-2yr-event-blog (#213)
chaos-mesh/website: blog/2021-12-27-chaos-mesh-story.md

author: Chaos Mesh Community
author_url: https://github.com/chaos-mesh
author_image_url: https://avatars1.githubusercontent.com/u/59082378?v=4
image: /img/chaos-mesh-story.jpeg
tags: [Chaos Mesh, community, event]
--- 

![Share your #ChaosMeshStory!](/img/chaos-mesh-story.jpeg)

Hey community,

🥳 Chaos Mesh turns 2 on 2021.12.31!  We're grateful for every contribution that helped this project grow. And we’d like to hear your Chaos Mesh story! How did you hear about the project? How did you get involved? Are you an adopter or a contributor? What do you think of it? It can be anything! Share your #ChaosMeshStory over on Twitter and win a Chaos Mesh Tee!

<!--truncate-->

## 🍼 #ChaosMeshStory Entry via Twitter 

### Rules

* Eligibility: anyone with a Twitter account
* Event Period: begins on December 27, 2021, at 9:00 AM PT and ends on December 31, 2022, at 11:59 PM PT
* How to participate:
    * Follow [@chaos_mesh](https://twitter.com/chaos_mesh).
    * Share your Chaos Mesh story on Twitter by using the **[#ChaosMeshStory](https://twitter.com/intent/tweet?text=%23ChaosMeshStory)** tag.
    * Share your experience with Chaos Mesh!

> * Judging Criteria: all qualified entries will be eligible to receive a Chaos Mesh Tee. The link to claim your SWAG will be published on the [@chaos_mesh](https://twitter.com/chaos_mesh) Twitter account after the event period is over.

Reviewer: ```suggestion
* Judging Criteria: all qualified entries will be eligible to receive a Chaos Mesh Tee. The link to collect your contact information will be available on January 1st, 2022. Stay tuned with [@chaos_mesh](https://twitter.com/chaos_mesh) on Twitter!
```

* Rules: please comply with the [code of conduct](https://github.com/chaos-mesh/chaos-mesh/blob/master/CODE_OF_CONDUCT.md), otherwise you will be ineligible to participate.

> * If you have any questions regarding the event, please dm [@chaos_mesh](https://twitter.com/chaos_mesh).
> 

Reviewer: ```suggestion

If you have any questions regarding the event, please dm [@chaos_mesh](https://twitter.com/chaos_mesh).

```

---
slug: /share-your-chaos-mesh-story
title: '[Special Event] Share your #ChaosMeshStory!'
author: Chaos Mesh Community
author_url: https://github.com/chaos-mesh
author_image_url: https://avatars1.githubusercontent.com/u/59082378?v=4
image: /img/chaos-mesh-story.jpeg
tags: [Chaos Mesh, community, event]
--- 

![Share your #ChaosMeshStory!](/img/chaos-mesh-story.jpeg)

Hey community,

> 🥳 Chaos Mesh turns 2 on 2021.12.31! We're grateful for every contribution that helped this project grow. And we'd like to hear your Chaos Mesh story! How did you hear about the project? How did you get involved? Are you an adopter or a contributor? What do you think of it? It can be anything! Share your #ChaosMeshStory over on Twitter and win a Chaos Mesh Tee!

Reviewer 2: ```suggestion
🥳 Chaos Mesh will turn 2 on 2021.12.31! We're grateful for every contribution from you that helped this project grow. And we'd like to hear your Chaos Mesh story! How did you hear about the project? How did you get involved? Are you an adopter or a contributor? What do you think of it? It can be anything! Share your #ChaosMeshStory over on Twitter and win a Chaos Mesh Tee!
```

<!--truncate-->

## 🍼 #ChaosMeshStory Entry via Twitter 

### Rules

* Eligibility: anyone with a Twitter account
* Event Period: December 27, 2021, at 9:00 AM - December 31, 2021, at 11:59 PM (PT)
* How to participate:
    1. Follow [@chaos_mesh](https://twitter.com/chaos_mesh).
    2. Add the **[#ChaosMeshStory](https://twitter.com/intent/tweet?text=%23ChaosMeshStory)** hashtag.
    3. Share your experience with Chaos Mesh!
* Judging Criteria: all qualified entries will be eligible to receive a Chaos Mesh Tee. The link to collect your contact information will be available on January 1st, 2022. Stay tuned with [@chaos_mesh](https://twitter.com/chaos_mesh) on Twitter!

> * Rules: please comply with the [code of conduct](https://github.com/chaos-mesh/chaos-mesh/blob/master/CODE_OF_CONDUCT.md), otherwise you will be ineligible to participate.

Reviewer 2: ```suggestion
* Rules: please comply with the [Code of Conduct](https://github.com/chaos-mesh/chaos-mesh/blob/master/CODE_OF_CONDUCT.md), otherwise you will be ineligible to participate.
```

> If you have any questions regarding the event, please dm [@chaos_mesh](https://twitter.com/chaos_mesh).

Reviewer 2: ```suggestion
If you have any questions regarding the event, please DM [@chaos_mesh](https://twitter.com/chaos_mesh).
```
